cmd/aws-cloud-controller-manager/main.go

```diff
@@ -109,6 +109,11 @@ func main() {
 				}
 			}
 
+			cloudConfigFlag := cmd.Flags().Lookup("cloud-config")
+			if cloudConfigFlag.Value.String() == "" {
```

Comment:
I _think_ that the cloud config file would actually be available as `c.ComponentConfig.KubeCloudShared.CloudProvider.CloudConfigFile` after we run `s.Config()` on line 119.

Reply:
so no need to use `cmd` to look up the flag value.

Reply:
I see!